gtk4/src/accessible_text_range.rs

```diff
@@ -1,17 +1,24 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-#[derive(Copy, Clone)]
-#[doc(alias = "GtkAccessibleTextRange")]
-#[repr(transparent)]
-pub struct AccessibleTextRange(crate::ffi::GtkAccessibleTextRange);
+use glib::translate::UnsafeFrom;
+use gtk4_sys::GtkAccessibleTextRange;
```

Comment:
You import it, then only use it in certain places... Usually we do `use crate::ffi;` and call ffi types with `ffi::$Type` or functions as well.